### PR TITLE
Improved basics-install docs to provide clarity.

### PR DESCRIPTION
**Pull request by Roseline-Bassey**
Addresses odk-x/tool-suite-X#  https://github.com/odk-x/tool-suite-X/issues/408

#### What is included in this PR?
Improved a sentence in the "Required" section of "Installing ODK-X Basic Tools" because the sentence may be unclear or hard to follow.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3.9.16-buster
```
Comment (elmps2018) [inline Dockerfile:1]: @wbrunette please check this change on python version.

**Review by elmps2018 — COMMENTED** [review]
@Roseline-Bassey  change to basics-install looks good, just want @wbrunette to check the python change.

**Review by wbrunette — APPROVED** [review]
